lab.R

```diff
@@ -110,32 +127,39 @@ daily_diff = function(x, order_var) {
 #' 1. *To explore these time-series data visually, we'll want to use line plots of cases or deaths over time.  The line group needs the `group` aesthetic to determine which values should be treated as part of a single line.  Uncomment and fill in the blanks to plot cases per 1,000,000 residents over time for each county.*  
 #' 
 
-# ggplot(covid_df, aes(---, ---, group = ---)) +
-#     geom_line()
+cases_plot = ggplot(covid_df, aes(date, cases_per_pop, group = county)) +
+    geom_line()
 
 #' 2. *Because there are so many counties, the lines are heavily overplotted.  Modify your code from the last problem to facet by county.  Try both `scales = 'fixed'` and `scales = 'free_y'`.* 
+# cases_plot + facet_wrap(vars(covid_df$county), scales = 'fixed')
+
+cases_plot = cases_plot + facet_wrap(vars(county), scales = 'free_y')
 #' 
 
 #' 3. *The plot indicates that, on a few days, some counties gain or lose thousands of cases per million residents.  What's up this that?  Hints:  `plotly::ggplotly()` to create an interactive version of the most recent plot.  Use `filter()` to narrow down the data to particular counties during short periods of time, and `View()` to peruse the data after filtering.*  
+plotly::ggplotly()
+
+kings_covid = filter(covid_df, county == 'Kings', date >= '2020-08-01' & date <= '2020-08-31')
+View(kings_covid)
```

Comment:
`View()` only makes sense when the code is run interactively; but Travis runs it automatically.  Try commenting this out.
```suggestion
# View(kings_covid)
```

Reply:
With a little further testing I'm pretty sure this is the problem.

Reply:
Oh okay, I tried commenting it out. I hope it works!